src/python/WMComponent/DBS3Buffer/MySQL/DBSBufferFiles/AddRunLumi.py

```diff
@@ -36,7 +36,8 @@ def getBinds(self, filename=None, runs=None):
                 for lumi in run:
```

Comment:
It does have lumis, but that only gets you the list of lumi numbers, not the event counts.